src/main/java/com/bookbla/americano/domain/member/repository/MemberEmailRepository.java

```diff
@@ -2,13 +2,19 @@
 
 import com.bookbla.americano.domain.member.repository.entity.Member;
 import com.bookbla.americano.domain.member.repository.entity.MemberEmail;
+import feign.Param;
+import java.time.LocalDateTime;
 import java.util.Optional;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
 
 public interface MemberEmailRepository extends JpaRepository<MemberEmail, Long> {
 
-    Optional<MemberEmail> findBySchoolEmail(String SchoolEmail);
-
     Optional<MemberEmail> findByMember(Member member);
+
+    @Modifying(clearAutomatically = true)
```

Comment:
```suggestion
    @Modifying(clearAutomatically = true, flushAutomatically = true)
```
저도 이번에 얼마전에 알게 된건데 [이런 이슈가 있네요  ](https://github.com/spring-projects/spring-data-jpa/issues/1167)
안전하게 flushAutomatically도 추가하면 어떨까요?!

Reply:
🤔 오호 그렇군요 바로 고치겠습니다